modules/features/endofyear/src/main/java/au/com/shiftyjelly/pocketcasts/endofyear/stories/Story.kt

```diff
@@ -0,0 +1,9 @@
+package au.com.shiftyjelly.pocketcasts.endofyear.stories
+
+import androidx.compose.ui.graphics.Color
+import au.com.shiftyjelly.pocketcasts.utils.seconds
+
+abstract class Story {
+    val storyLength: Long = 2.seconds()
```

Comment:
Story length is currently the same for all stories. It'll be made dynamic in a future PR.